mithril.js

```diff
@@ -1,5 +1,4 @@
 new function() {
-"use strict"
```

Comment:
Yuck...could you make sure that remains somehow? Mithril shouldn't be running in sloppy mode.